recipes/assemblyutility/meta.yaml

```diff
@@ -0,0 +1,34 @@
+{% set name = "AssemblyUtility" %}
+{% set version = "20160209" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+build:
+  number: 0
+
+source:
+  git_url: https://github.com/yechengxi/AssemblyUtility.git
```

Comment:
Yes please, we recommend to use tarball and not git checkouts.
For your other error wich you see, please always create new branches from latest master, or rebase against latest master.